Browser_IDE/executionEnvironment.js

```diff
@@ -216,6 +216,12 @@ class ExecutionEnvironment extends EventTarget{
             newPath: newPath,
         });
     }
+    async readFile(path){
+        this.iFrame.contentWindow.postMessage({
+            type: "readFile",
+            path: path
+        }, "*");
```

Comment:
Right now this just sends a message, but has no way to get the returned value. Use `postMessageFallible` instead of `postMessage`, and make sure to `return` the result.